.dockerignore

```diff
@@ -1,11 +1,14 @@
 # Blanket ignore everything by default
 *
+cdk*
 
 # Whitelist files and folders that should be in the container
 !app
 !migrations
 !scripts
 
+!phuks.txt
+!config.yaml
```

Comment:
Just want to make sure people who use this understand that there is a possibility of leading to a situation where your sensitive values are hard-baked into a Docker image. Must use with caution.